Uppsala/UppEt3.xml

```diff
@@ -0,0 +1,557 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt3">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">
+                    Revelation of John, Maṣḥafa Ḥassāb
+                </title>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 3</idno>
+                        <altIdentifier><idno>Löfgren 3</idno></altIdentifier>
```

Comment:
I assume the link to imges should be included.

Reply:
I included it at the very end of the file, from line 549 and forward. Should they be placed here?

Reply:
I think so; this is the place to refer to the images online (below you refer to the description in the database), the link may be `http://urn.kb.se/resolve?urn=urn:nbn:se:alvin:portal:record-177135` (Zhenia might know better which exactly you pick).

Reply:
`Facsimile` graphic url is the correct place for external links ( permalinks)

`Facs` is only for iiif manifests when these have been previously enabled.